### DISCUSSION TITLE
Codespaces (web) on iPad - a few usability speed bumps

### DISCUSSION THREAD

**Discussion by dzsquared** [community/community#5390] [Codespaces]
(padOS 15 public beta 7)
There are some components in the web interface that don’t interact as I expect on an iPad when using an external mouse/trackpad and external keyboard.
1. Scrolling - the activity pane on the left, if the list of files is expanded such that scrolling is needed - scrolling can only be done through touch and cannot be done by scrolling the mouse/trackpad.
2. Text input - It is common to click into a pane (source control, for example) and see the commit message input highlighted.  However typing does not render text in the box - generally hitting tab, then shift tab to go back to the text input corrects the issue.  (same occurs on search box)
3. Text editor copy/paste.  Often when the cursor is in the text editor and a keyboard shortcut is used to paste - the paste action inserts text into the terminal instead of the text editor.  Right-clicking and selecting paste in the text editor does insert the text into the correct location - but the “paste” bubble appears in the upper left corner and must be clicked before the text appears.

Codespaces in the web has been a game changer for working from the iPad and have loved using it!

**Comment by jozefvaclavik** [comment]
For me scrolling used to work on first few betas (afaik up to public beta 2)

**Comment by bamurtaugh** [comment]
Thanks for opening @dzsquared, and so glad you're enjoying Codespaces on the iPad!

We have an `ios-ipados` label in the VS Code GitHub repo to [track iPad-related issues](https://github.com/microsoft/vscode/issues?q=is%3Aissue+is%3Aopen+label%3Aios-ipados+). I think a good next step might be reporting your issues there (in separate issues) to raise awareness and so that the team can investigate appropriately.
